src/apps/checklist/checklist.entry.jsx

```diff
@@ -91,18 +105,24 @@ function ChecklistEntry({ materialUrl, authorUrl, removeButtonText }) {
       materialUrl={materialUrl}
       authorUrl={authorUrl}
       removeButtonText={removeButtonText}
+      emptyListText={emptyListText}
+      errorText={errorText}
     />
   );
 }
 
 ChecklistEntry.propTypes = {
   materialUrl: PropTypes.string.isRequired,
   authorUrl: PropTypes.string.isRequired,
-  removeButtonText: PropTypes.string
+  removeButtonText: PropTypes.string,
+  emptyListText: PropTypes.string,
+  errorText: PropTypes.string
 };
 
 ChecklistEntry.defaultProps = {
-  removeButtonText: "Fjern fra listen"
+  removeButtonText: "Fjern fra listen",
+  emptyListText: "Listen er tom",
+  errorText: "Noget git galt"
```

Comment:
```suggestion
  errorText: "Noget gik galt"
```

In general: Should these texts be the same as in `checklist.dev.jsx` or is it intentional that they are different?

Reply:
Kinda intentional so that we can see that there actually is a difference between defaultProps and what is passed to the component.